customwidgets/_slider-theme.scss

```diff
@@ -0,0 +1,53 @@
+.widget-slider {
+    .rc-slider-handle, .rc-slider-dot-active {
+        border: 2px solid $brand-default;
```

Comment:
Please split this property. move the `border: 2px solid;` to your static styling and keep
`border-color: $brand-default;`